inst/sql/sql_server/concept_plausible_gender.sql

```diff
@@ -23,6 +23,7 @@ FROM
 	SELECT COUNT_BIG(*) AS num_violated_rows
 	FROM
 	(
+		/*violatedRowsBegin*/
```

Comment:
Are these tags (violatedRowsBegin/End) for new functionality of the v2 viewer?

Ok for me to keep this in, although it has no added value within the context of this PR.

Reply:
If there is going to be a v2 viewer similar to what has been presented then those tags are necessary for a new functionality . Check comments below.